packages/omsorgspenger-omsorgen-for/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "@k9-sak-web/omsorgspenger-omsorgen-for",
```

Comment:
vi har pleid å navngi pakker med typen til slutt, altså "omsorgen-for-oms" eller "omsorgen-for-omsorgspenger"

Reply:
Renamet til "omsorgen-for-oms" 👍